src/Native/Unix/System.Security.Cryptography.Native/openssl.c

```diff
@@ -1239,11 +1221,6 @@ static int32_t EnsureOpenSsl10Initialized()
     // Initialize the callback
     CRYPTO_set_locking_callback(LockingCallback);
 
-#ifdef __APPLE__
-    // OSX uses an earlier version of OpenSSL which requires setting the CRYPTO_set_id_callback
-    CRYPTO_set_id_callback(GetCurrentThreadId);
```

Comment:
Don't we still need this when running against openssl 1.0.x?

Reply:
No, that was apparently left over from 0.9.8 :smile: